pkg/ocm/clusters.go

```diff
@@ -588,6 +588,13 @@ func (c *Client) UpdateCluster(clusterKey string, creator *aws.Creator, config S
 		clusterBuilder = clusterBuilder.ExpirationTimestamp(config.Expiration)
 	}
 
+	// Update channel group
+	if config.ChannelGroup == "" {
```

Comment:
Why send when empty?

Reply:
leftover mistake I guess- it worked though surprisingly lol